packages/wdio/src/component/component/component.ts

```diff
@@ -65,15 +66,15 @@ export class Component {
         return await (await this.getChildEl(selector)).getSize();
     }
 
-    async getChildEl(selector: string): Promise<WebdriverIO.Element> {
-        return await this.elSelectorFn(selector);
+    getChildEl(selector: string): ChainablePromiseElement<WebdriverIO.Element> {
+        return this.elSelectorFn(selector) as ChainablePromiseElement<WebdriverIO.Element>;
```

Comment:
Why not change the type of `elSelectorFn` to return `ChainablePromiseElement`

Reply:
Ohhh, this brings huge amount of unfixable TS issues